### ISSUE TITLE
[Idea] Can you add a tutorial for installing Pterodactyl on Plesk

### ISSUE THREAD

**Issue by ZeProf2Code** [pterodactyl/documentation#54]
Everything is in the title,

I use Debian 8.11

**Comment by ZeProf2Code** [comment]
If anyone ever knows how to do it, I'll take

**Comment by lancepioch** [comment]
We appreciate the thoughts of getting Pterodactyl working on more platforms, but hosting panels typically introduce many more problems with hosting full fledged web applications like this one.

Some of the typical issues:

* don't always have ssh access
* can't always control php/mysql versions
* can't always control php extensions
* usually can't access supervisor or systemd
* if running on the same system, node.js/go support for wings

Simply put, if you're unable to install Pterodactyl roughly using the general instructions (for most linux servers), then it'll most likely be an extra patch of headaches that we won't be able to support.
